src/configs/buf.rs

```diff
@@ -20,6 +21,27 @@ pub struct BufConfig<'a> {
 
 impl<'a> Default for BufConfig<'a> {
     fn default() -> Self {
+        if cfg!(windows) {
```

Comment:
Would it be better then if I replaced the Windows 11 check operation with replacing the bison emoji with the water buffalo instead since it is more available?

Reply:
I'm in favor of that change.